demo/portal/src/app/services/userService.js

```diff
@@ -41,6 +52,12 @@ UserService.prototype.logout = function logout() {
 }
 
 
-angular.module('wfm.common.apiservices').service("userService", ['authService', function(authService) {
-  return new UserService(authService);
+angular.module('wfm.common.apiservices').service("userService", ['authService', '$http', function(authService, $http) {
+  var urlPromise = new Promise(function(resolve, reject) {
```

Comment:
Let's keep it that way. There is this PR for SDK that will allow us to initialize SDK when we need to so we can wrap that into service.